cloud/amazon/cloudformation.py

```diff
@@ -14,12 +14,22 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
+# upcoming features:
```

Comment:
I can remove them, but as soon as this is pulled I'll add those features. I'm trying to keep this one as the baseline before making further changes. Let me know.